k8s/elasticsearch/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,72 @@
 actions:
-- name: dummy test
+
+- name: Wait until Elasticsearch is ready
   bashTest:
     script: |-
-      exit 0
+      for ((pod=0; pod<${REPLICAS}; pod++)); do
+        kubectl wait pod/${APP_INSTANCE_NAME}-elasticsearch-${pod} \
+          --namespace=${NAMESPACE} \
+          --for=condition=Ready \
+          --timeout=180s
+      done
     expect:
       exitCode:
         equals: 0
+
+- name: All nodes are connected to the cluster
+  bashTest:
+    script: |-
+      nodes="$(curl -XGET "${HEALTH_URL}?wait_for_nodes=${REPLICAS}&timeout=300s" | jq '.number_of_nodes')"
+      if [[ "${nodes}" -eq "${REPLICAS}" ]]; then
+        echo 'OK - all nodes found'
+      else
+        echo "FAIL - found nodes: ${nodes}"
+      fi
+    expect:
+      stdout:
+        equals: 'OK - all nodes found'
```

Comment:
JFYI: Because of https://github.com/GoogleCloudPlatform/marketplace-testrunner/pull/6 change, we will be able to drop the if statement and replace equals with something like `{ .Env.REPLICAS }`